src/internal/views/nunjucks/billing/two-part-tariff-review.njk

```diff
@@ -0,0 +1,89 @@
+{% extends "./nunjucks/layout.njk" %}
+{% from "badge.njk" import badge %}
+{% from "paginate.njk" import paginate %}
+{% from "./nunjucks/billing/macros/batch-metadata.njk" import batchMetadata %}
+{% from "govuk/components/button/macro.njk" import govukButton %}
+{% from "./nunjucks/billing/macros/totals.njk" import chargeOrDash %}
+
+{% block content %}
+<div class="govuk-grid-row govuk-!-margin-bottom-0">
+  <div class="govuk-grid-column-two-thirds">
+    <section class="govuk-!-margin-bottom-6">
+      {% set caption %}
+        {{ 'Bill run ' + batch.externalId if batch.externalId else null }}
+      {% endset %}
+        {{ title(pageTitle, 'This is the caption', null, true, true) }}
+    </section>
+  </div>
+</div>
+<div class="govuk-grid-row govuk-!-margin-bottom-6">
+<div class="govuk-grid-column-two-thirds">
+      <p class="govuk-body">
+        {{ badge(batch | batchBadge(false)) }}
+      </p>
+      <div class="govuk-grid-row">
+        {{ batchMetadata(batch) }}
+      </div>
+  </div>
+</div>
+ 
+<section class="govuk-!-margin-bottom-6">
+  {{
+    govukButton({
+      text: "Cancel bill run",
+      href: '/billing/batch/' + batch.id + '/cancel',
+      classes: "govuk-button--secondary govuk-!-margin-left-1"
+    })
+  }}
+</section>
+
+<section class="govuk-!-margin-bottom-9">
+  <h2 class="govuk-heading-l" id="issuesTitle">{{ totals.errors }} licences have data issues</h2>
+  <div class="table govuk-!-margin-bottom-6" id="dataIssues">
+    {% if licences.length > 0 %}
```

Comment:
I think this if statement should move up a couple of lines so that this whole block titled:
"X licences have data issues"
does not display when all issues have been resolved

Reply:
done